mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapperTest.java

```diff
@@ -75,6 +76,7 @@ public void saveShouldNotRemoveOldMailboxPathWhenCreatingTheNewMailboxPathFails(
             testee.save(newMailbox);
             fail("TooLongMailboxNameException expected");
         } catch (TooLongMailboxNameException e) {
+            LOGGER.info("Ignored error", e);
```

Comment:
I can not as the exception thrown would prevent latest test instructions to be executed.